static/redfish/v1/$metadata/index.xml

```diff
@@ -1411,6 +1411,19 @@
     <edmx:Reference Uri="/redfish/v1/schema/LogServiceCollection_v1.xml">
         <edmx:Include Namespace="LogServiceCollection"/>
     </edmx:Reference>
+
+    <edmx:Reference Uri="/redfish/v1/schema/LicenseService_v1.xml">
```

Comment:
I think you need to rebase this? These should have already been added as part of 2021.3

Reply:
Actually I see them already 

    <edmx:Reference Uri="/redfish/v1/schema/License_v1.xml">
        <edmx:Include Namespace="License"/>
        <edmx:Include Namespace="License.v1_0_0"/>
    </edmx:Reference>
    <edmx:Reference Uri="/redfish/v1/schema/LicenseCollection_v1.xml">
        <edmx:Include Namespace="LicenseCollection"/>
    </edmx:Reference>
    <edmx:Reference Uri="/redfish/v1/schema/LicenseService_v1.xml">
        <edmx:Include Namespace="LicenseService"/>
        <edmx:Include Namespace="LicenseService.v1_0_0"/>
    </edmx:Reference>
    
    So this can be remove

Reply:
Deleted lines